modules/Microsoft.ContainerService/managedClusters/deploy.bicep

```diff
@@ -295,6 +295,20 @@ param tags object = {}
 @description('Optional. The resource ID of the disc encryption set to apply to the cluster. For security reasons, this value should be provided.')
 param diskEncryptionSetID string = ''
 
+@description('Optional. ReleaseTrain this extension participates in for auto-upgrade (e.g. Stable, Preview, etc.) - only if autoUpgradeMinorVersion is "true".')
```

Comment:
I'd suggest to include all parameters related to the newly cross referenced module in a single object.

Reply:
I hope the user-definied types will allows us to specify individual 'secure' values one released. Similar to TF.

Reply:
This was updated to now be in the `fluxExtension` parameter.